imap_processing/tests/conftest.py

```diff
@@ -119,9 +142,16 @@ def get_test_kernels_to_load():
                         kernel = kernel[stop_idx:]
         return kernels_to_load
 
-    metakernel_path = imap_data_access.config["DATA_DIR"] / "imap_2024_v001.tm"
+    metakernel_path = tmpdir_factory.mktemp("spice") / "imap_2024_v001.tm"
     kernels_to_load = get_test_kernels_to_load()
     make_metakernel_from_kernels(metakernel_path, kernels_to_load)
-    monkeypatch.setenv("SPICE_METAKERNEL", str(metakernel_path))
+    monkeypatch_session.setenv("SPICE_METAKERNEL", str(metakernel_path))
     yield str(metakernel_path)
     spice.kclear()
+
+
+@pytest.fixture()
+def _unset_metakernel_path(monkeypatch):
+    """Temporarily unsets the SPICE_METAKERNEL environment variable"""
+    if os.getenv("SPICE_METAKERNEL", None) is not None:
+        monkeypatch.delenv("SPICE_METAKERNEL")
```

Comment:
Looks like you can avoid the if check with a raising=False keyword.
https://docs.pytest.org/en/stable/reference/reference.html#pytest.MonkeyPatch.delenv
```suggestion
    monkeypatch.delenv("SPICE_METAKERNEL", raising=False)
```